### DISCUSSION TITLE
Capterra.com is detecting the webdriver

### DISCUSSION THREAD

**Discussion by Leoplazavzla** [ultrafunkamsterdam/undetected-chromedriver#1207] [General]
The website Capterra.com still detects the webdriver and prevents the loading of new information. 
I've tried changing the User Agent and adding cookies from a session on Chrome but still detects it quickly

**Comment by y0un9kane** [comment]
I have successfully opened and loaded "Capterra.com" using "undetected_chromedriver" without any issues.

**Reply by Leoplazavzla** [reply]
Yeah, I did open the website and everything. But try scraping at least 100 records.
It will block your access to it

**Comment by y0un9kane** [comment]
I think you should look for reasons from the frequency of clicks, IP changes, these places
